Source/NSSpeechRecognizer.m

```diff
@@ -23,8 +23,174 @@
 */
 
 #import <AppKit/NSSpeechRecognizer.h>
+#import <Foundation/NSDistantObject.h>
+#import <Foundation/NSString.h>
+#import <Foundation/NSDictionary.h>
+#import <Foundation/NSArray.h>
+#import <Foundation/NSThread.h>
+#import <Foundation/NSError.h>
+#import <Foundation/NSConnection.h>
+#import <Foundation/NSDistributedNotificationCenter.h>
 
-@implementation NSSpeechRecognizer
+#import "AppKit/NSWorkspace.h"
+
+id   _speechRecognitionServer = nil;
+BOOL _serverLaunchTested = NO;
+
+#define SPEECH_RECOGNITION_SERVER @"GSSpeechRecognitionServer"
 
+@interface NSObject (GSSpeechRecognitionServer)
+- (NSSpeechRecognizer *) newRecognizer;
 @end
 
+@implementation NSSpeechRecognizer
+
++ (void) initialize
+{
+  _speechRecognitionServer = [NSConnection
```

Comment:
This part will get executed in the subclass GSSpeechRecognizer as well. Is this intentional?

Reply:
No, I don't think it is...

Reply:
Fixed.